modules/psap-node-feature-discovery-topology-updater-command-reference.adoc

```diff
@@ -0,0 +1,190 @@
+
+[IMPORTANT]
+====
+The `-cert-file` flag must be specified together with the `-ca-file` and `-key-file` flags.
+====
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -cert-file=/opt/nfd/updater.crt -key-file=/opt/nfd/updater.key -ca-file=/opt/nfd/ca.crt
+----
+
+[discrete]
+[id="nfd-topology-updater-key-file_{context}"]
+== -key-file
+
+The `-key-file` flag is one of the three flags, together with `-ca-file` and `-cert-file`, controlling the mutual TLS authentication on the NFD Topology Updater. This flag specifies the private key corresponding the given certificate file, or `-cert-file`, that is used for authenticating outgoing requests.
+
+Default: empty
+
+[IMPORTANT]
+====
+The `-key-file` flag must be specified together with the `-ca-file` and `-cert-file` flags.
+====
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -key-file=/opt/nfd/updater.key -cert-file=/opt/nfd/updater.crt -ca-file=/opt/nfd/ca.crt
+----
+
+[discrete]
+[id="nfd-topology-updater-server-name-override_{context}"]
+== -server-name-override
+
+The `-server-name-override` flag specifies the common name (CN) which to expect from the nfd-master TLS certificate. This flag is mostly intended for development and debugging purposes.
+
+Default: empty
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -server-name-override=localhost
+----
+
+[discrete]
+[id="nfd-topology-updater-no-publish_{context}"]
+== -no-publish
+
+The `-no-publish` flag disables all communication with the nfd-master, making it a "dry-run" flag for nfd-topology-updater. NFD Topology Updater resource hardware topology detection normally, but no CR requests are sent to nfd-master.
+
+Default: `false`
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -no-publish
+----
+
+[discrete]
+[id="nfd-topology-updater-oneshot_{context}"]
+== -oneshot
+
+The `-oneshot` flag causes the NFD Topology Updater to exit after one pass of resource hardware topology detection.
+
+Default: `false`
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -oneshot -no-publish
+----
+
+[discrete]
+[id="nfd-topology-updater-sleep-interval_{context}"]
+== -sleep-interval
+
+The `-sleep-interval` flag specifies the interval between resource hardware topology re-examination and custom resource updates. A non-positive value implies infinite sleep interval and no re-detection is done.
+
+Default: `60s`
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -sleep-interval=1h
+----
+
+[discrete]
+[id="nfd-topology-updater-watch-namespace_{context}"]
+== -watch-namespace
+
+The `-watch-namespace` specifies the namespace to ensure that resource hardware topology examination only happens for the pods running in the
+specified namespace. Pods that are not running in the specified namespace are not considered during resource accounting. This is particularly useful for testing and debugging purposes. A `*` value means that all of the pods are considered during the accounting process.
```

Comment:
For the last sentence, does this mean all of the pods across all namespaces? (If so, should that be clarified?)